doc/man3/flux_open.rst

```diff
@@ -42,6 +42,11 @@ FLUX_O_MATCHDEBUG
 FLUX_O_NONBLOCK
    The ``flux_send()`` and ``flux_recv()`` functions should never block.
 
+FLUX_O_TEST_NOSUB
```

Comment:
commit message typo: "flag is not document." :arrow_right: "flag is not documented."